sqlmesh/utils/formatting.py

```diff
@@ -0,0 +1,3 @@
+def indent(string: str, num: int = 1) -> str:
```

Comment:
why do you need this?

Reply:
Its difficult to scan the log when the query is inline with all the other text. Indenting made it easy to parse.

Reply:
this only indents line breaks though, can you just do

query.sql(indent=4)

Reply:
Didn't realize sql() had an indent option. Will look into that.

Reply:
@tobymao what I'm trying to do is indent level 0. `indent` only applies to level 1+.

Reply:
![Screenshot 2023-05-31 at 9 51 26 PM](https://github.com/TobikoData/sqlmesh/assets/67359577/c63ce837-dabe-414f-a4dd-9eddc49c3776)

Reply:
ok, use textwrap.indent